### PR TITLE
Add initial CI with prettier check, package.json

### PR DESCRIPTION
**Pull request by estherkim**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  pull_request:
+    branches:
+      - main
+
+  push:
+    branches:
+      - main
+
+  workflow_dispatch:
```
Comment (samouri) [inline .github/workflows/ci.yml:12]: what is this line? it is bare which I find suspicious.

Comment (mdmower) [reply .github/workflows/ci.yml:12]: It's legit. It allows a GitHub workflow to be triggered manually and that workflow does not require any inputs.

Comment (estherkim) [reply .github/workflows/ci.yml:12]: haha. it's to trigger manually

Comment (estherkim) [reply .github/workflows/ci.yml:12]: oops just saw your comment @mdmower. long time, hi!!

**Review by rileyajones — APPROVED** [review]

**Review by samouri — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (samouri) [reply .github/workflows/ci.yml:12]: Hey Matt! Thanks for the info & hope all is well. Feel free to drive by with helpful comments whenever you like 😄

**Pull request merged**